main.nf

```diff
@@ -118,13 +118,6 @@ workflow NFCORE_PHAGEANNOTATOR {
         ch_multiqc_logo.toList()
     )
     multiqc_report = MULTIQC.out.report.toList()
-
-    emit:
-    filtered_viruses_fna_gz     = PHAGEANNOTATOR.out.filtered_viruses_fna_gz
```

Comment:
why removing the emits ?

Reply:
Earlier on I was experimenting with using them for the pipeline nf-tests, but workflow emits can't be used in pipeline nf-tests. I removed them because they're not being used